boilerplate/frontend/react/src/App.css

```diff
@@ -0,0 +1,276 @@
+@font-face {
```

Comment:
The vertical scrollbar appears on the auth page.
The topbar links could look nicer/fit a bit better into the theme.